webserver/static/scripts/datasets/eval-jobs-viewer.js

```diff
@@ -89,13 +105,34 @@ var EvaluationJobsViewer = React.createClass({
         });
         window.location.hash = "";
     },
+    handleJobDelete: function (jobID, index) {
+        $.ajax({
+            type: "DELETE",
+            url: "/datasets/" + container.dataset.datasetId + "/" + jobID,
+            success: function (data, textStatus, jqXHR) {
+                alert("Evaluation job " + jobID + " has been removed from the queue.")
```

Comment:
It goes away from the list anyway. Perhaps we can hide it or add some other indication of the fact that it is being deleted (though this should be too fast to notice), then remove it completely on success or show error message next to it on failure.